site/source/tutorials/accessibility.md

```diff
@@ -0,0 +1,233 @@
+___
+
+### Checkbox
+The Checkbox widget wraps a native `<input type="checkbox">`, which provides native accessibility and keyboard interaction.
+
+Checkbox can also be used as a toggle switch with optional `onLabel` and `offLabel` properties. When no text is provided to the on- and off-labels, the change is purely cosmetic. Otherwise, the text provided to `onLabel` will be read at the end of the label text when the checkbox is checked, and `offLabel` will be read when the checkbox is not checked.
+
+##### A11y properties
+- `describedBy`: Sets the `aria-describedby` property to point to the ID of an element containing additional descriptive text. Screen readers usually read this text after the label text, followed by a short pause.
+- `label`: Controls the wrapping `<label>` element, and takes a string or options object
+___
+
+### Radio
+Radio creates a single wrapped `<input type="radio">` input. It is best practice to wrap a group of radio buttons in a `<fieldset>` element that contains a `<legend>` with descriptive text.
+##### A11y properties
+- `describedBy`: Sets the `aria-describedby` property to point to the ID of an element containing additional descriptive text. Screen readers usually read this text after the label text, followed by a short pause.
+- `label`: Controls the wrapping `<label>` element, and takes a string or options object
+___
+
+### Slider
+A native range slider offers limited visual customization, so the Dojo Slider widget creates an invisible native `<input type="range">` input that controls a custom track, thumb, and fill. This allows the widget to take advantage of native accessibility and keyboard interaction without sacrificing styling.
+
+The current value is wrapped in an `<output>` element that points to the ID of the native input.
+
+##### A11y properties
+- `describedBy`: Sets the `aria-describedby` property to point to the ID of an element containing additional descriptive text. Screen readers usually read this text after the label text, followed by a short pause.
+- `label`: Controls the wrapping `<label>` element, and takes a string or options object
+___
+
+### Select
+Select is the only Dojo form widget that makes use of a fully custom solution. Since it is impossible to fully customize `<option>` elements, we chose to re-create select functionality within our component. The end result is a popup button that controls a dropdown listbox of options.
+
+ARIA attributes are used to associate the button with the listbox and focused option. They are also used to provide semantics for read-only and disabled options, as well as setting the entire select field to be read-only, invalid, or required.
+
+It is also possible to use Select to create a widget using the native `<select>` element by passing `useNativeElement: true` to the properties object.
+
+##### Keyboard Interaction
+- **Down arrow**: Focuses the next option, wrapping from last to first, or opens the menu if closed
```

Comment:
Should this say "opens the listbox if closed" to be consistent with the other keyboard interaction descriptions?  "menu" isn't used in any other part of the Select section.  But, this section uses "dropdown listbox" and the next ComboBox section uses "option menu" and "menu comprises of a listbox of options".  The two sections need to be made consistent because the two widgets are not that much different.

Reply:
Updated to use "menu" over "listbox" except when specifically referencing the role, since I think "menu" will be the more familiar term.